code/core/arm9/source/Application/GbaDisplayConfigurationService.cpp

```diff
@@ -125,10 +126,17 @@ void GbaDisplayConfigurationService::SetupGbaScreen(const DisplaySettings& displ
 
 void GbaDisplayConfigurationService::SetupColorCorrection(const DisplaySettings& displaySettings)
 {
-    if (displaySettings.gbaColorCorrection == GbaColorCorrection::None)
-    {
-        clut_disableColorCorrection();
-    }
+    const auto getEnum = displaySettings.gbaColorCorrection;
+        const ColorProfile* profile = colorProfileLut[static_cast<size_t>(getEnum)];
+        if (profile)
```

Comment:
Please use { and } for all if and else blocks. It improves readability of the code